docs/upgrade-3-4.md

```diff
@@ -266,6 +219,81 @@ tail -f /var/log/armadillo/*
 
 ## 6. Visit the site
 
-If all went well you now have a new version installed.
+Go to your armadillo website. Is the version in the left top corner updated? This means the update was successful. We're
+almost finished. 
+
+## 7. Update profiles
+Login into the website and go to the profiles tab. Here two profiles should be listed: `default` and `xenon`. 
+
+1. Stop the xenon profile. 
+2. Stop the default profile.
+3. Edit the default profile. 
+4. Change the "image" to `datashield/rock-base:latest` and save. 
+5. Start the default profile.
+6. Edit the "xenon" profile.
+7. Change the "image" to `datashield/rock-dolomite-xenon:latest` and save.
+8. Start the xenon profile. 
```

Comment:
They are stopped and changed in stap 2.2 through UI and removed by docker commands.

Reply:
You're right! I've changed it